src/main/java/com/revellr/core/partyservice/DataLayer/UserInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.revellr.core.partyservice.DataLayer;
+
+import java.util.*;
+import com.revellr.core.partyservice.MysqlInterface.UserTable;
+
+
+
+public class UserInterface{
+
+    public String CreateUser(Map<String , String> input)
+    {
+        User newUser = new Models.User();
+        newUser.setName(input.firstname , input.lastname);
```

Comment:
preferably use Builder pattern for creating the user instance. @Builder annotation in lombok provides that.